### PR TITLE
Bot Card Design Implemented

### PR DESCRIPTION
**Pull request by srikanthv02**
@udeshrl : Please merge.

### CODE REVIEW

File: client/cat3/src/partials/sections/dashboard/bots/view/botHistory.html [hunk 1]
```diff
@@ -14,29 +14,29 @@ <h4 class="modal-title">
 			<!--This table will get use when bot job type will be chef -->
 			<loading size="large" name="ischefTaskHistoryPageLoading" type="block" align="center"></loading>
 			<div ng-show="!ischefTaskHistoryPageLoading">
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/bots/view/botHistory.html]: remove the ng-show condition . use only one UI grid for all history .

File: client/cat3/src/partials/sections/dashboard/bots/controller/botHistoryCtrl.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+                $scope.initChefGrids();
+                $scope.taskHistoryChefListView();
+                $scope.getExecutionTime();
+                $scope.getSavedTime();
+            };
+            //UI Grid for chef Task ends
+
+            //UI Grid for jenkins Task starts
+            $scope.initJenkinsGrids = function(){
+                $scope.taskHistoryJenkinsGridOptions={};
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/bots/controller/botHistoryCtrl.js]: use only one UI grid object for all type history  .

File: client/cat3/src/partials/sections/dashboard/bots/controller/botHistoryCtrl.js [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+            $scope.initblueprint = function(){
+                $scope.initBlueprintGrids();
+                $scope.botHistoryBlueprintListView();
+                $scope.getExecutionTime();
+                $scope.getSavedTime();
+            };
+            //UI Grid for Blueprint ends
+
+            $scope.bot=items;
+            switch ($scope.bot.botLinkedSubCategory){
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/bots/controller/botHistoryCtrl.js]: remove the switch case .

File: client/cat3/src/partials/sections/dashboard/bots/controller/botHistoryCtrl.js [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+
+            $rootScope.$on('BOTS_TEMPLATE_SELECTED', function(event,reqParams) {
+                $scope.templateSelected = reqParams;
+            });
+            
+            if($scope.templateSelected) {
+                items = $scope.templateSelected;
+            }
+        
+            $scope.botHistory = items;
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/bots/controller/botHistoryCtrl.js]: why you save "item" value in scope .

Comment (srikanthv02) [inline client/cat3/src/partials/sections/dashboard/bots/controller/botHistoryCtrl.js]: We need to show the item details in the html.

**Pull request merged**